llvm/test/CodeGen/AMDGPU/rewrite-vgpr-mfma-scale-to-agpr.mir

```diff
@@ -0,0 +1,6793 @@
+  - { id: 3509, class: vgpr_32, preferred-register: '', flags: [  ] }
+  - { id: 3510, class: sreg_64, preferred-register: '', flags: [  ] }
+  - { id: 3511, class: vgpr_32, preferred-register: '', flags: [  ] }
+  - { id: 3512, class: sgpr_32, preferred-register: '', flags: [  ] }
+  - { id: 3513, class: vgpr_32, preferred-register: '', flags: [  ] }
+  - { id: 3514, class: vgpr_32, preferred-register: '', flags: [  ] }
+  - { id: 3515, class: sgpr_32, preferred-register: '', flags: [  ] }
+  - { id: 3516, class: vgpr_32, preferred-register: '', flags: [  ] }
+  - { id: 3517, class: vreg_64_align2, preferred-register: '', flags: [  ] }
+  - { id: 3518, class: vgpr_32, preferred-register: '', flags: [  ] }
+  - { id: 3519, class: vgpr_32, preferred-register: '', flags: [  ] }
+  - { id: 3520, class: sgpr_128, preferred-register: '', flags: [  ] }
+  - { id: 3521, class: vreg_128_align2, preferred-register: '', flags: [  ] }
+  - { id: 3522, class: av_128_align2, preferred-register: '', flags: [  ] }
+  - { id: 3523, class: av_128_align2, preferred-register: '', flags: [  ] }
+  - { id: 3524, class: vgpr_32, preferred-register: '', flags: [  ] }
+  - { id: 3525, class: vreg_128_align2, preferred-register: '', flags: [  ] }
+  - { id: 3526, class: av_128_align2, preferred-register: '', flags: [  ] }
+  - { id: 3527, class: av_128_align2, preferred-register: '', flags: [  ] }
+  - { id: 3528, class: vgpr_32, preferred-register: '', flags: [  ] }
+  - { id: 3529, class: vreg_128_align2, preferred-register: '', flags: [  ] }
+  - { id: 3530, class: av_128_align2, preferred-register: '', flags: [  ] }
+  - { id: 3531, class: av_128_align2, preferred-register: '', flags: [  ] }
+  - { id: 3532, class: vgpr_32, preferred-register: '', flags: [  ] }
+  - { id: 3533, class: vreg_128_align2, preferred-register: '', flags: [  ] }
+  - { id: 3534, class: av_128_align2, preferred-register: '', flags: [  ] }
+  - { id: 3535, class: av_128_align2, preferred-register: '', flags: [  ] }
+  - { id: 3536, class: vgpr_32, preferred-register: '', flags: [  ] }
+  - { id: 3537, class: vreg_128_align2, preferred-register: '', flags: [  ] }
+  - { id: 3538, class: av_128_align2, preferred-register: '', flags: [  ] }
+  - { id: 3539, class: av_128_align2, preferred-register: '', flags: [  ] }
+  - { id: 3540, class: vgpr_32, preferred-register: '', flags: [  ] }
+  - { id: 3541, class: vreg_128_align2, preferred-register: '', flags: [  ] }
+  - { id: 3542, class: av_128_align2, preferred-register: '', flags: [  ] }
+  - { id: 3543, class: av_128_align2, preferred-register: '', flags: [  ] }
+  - { id: 3544, class: vgpr_32, preferred-register: '', flags: [  ] }
+  - { id: 3545, class: vreg_128_align2, preferred-register: '', flags: [  ] }
+  - { id: 3546, class: av_128_align2, preferred-register: '', flags: [  ] }
+  - { id: 3547, class: av_128_align2, preferred-register: '', flags: [  ] }
+  - { id: 3548, class: vgpr_32
```

Comment:
This needs to be a lot smaller. In particular you should drop the registers section and compact the register numbers